apps/ts-minbar-test-react-components/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ts-minbar-test-react-components",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Testing Fluent UI React Components compatibility with Typescript 3.9",
+  "license": "MIT",
+  "dependencies": {
+    "@fluentui/react-components": "^9.0.0-beta.3"
+  },
+  "scripts": {
+    "build": "just-scripts build",
```

Comment:
Less important but this should probably also have other common scripts such as lint and clean